src/components/Login.js

```diff
@@ -40,78 +42,85 @@ export class Login extends Component {
   }
 
   render() {
-    const { username, token} = this.state;
+    const { username, token } = this.state;
     const { loading } = this.props;
 
     return (
       <div className="signinModal">
-            <div className="modal-dialog" role="document">
-              <div className="modal-content ">
-                <div className="modal-header">
-                  <h3 className="modal-title">
-                    <strong>Login</strong>
-                  </h3>
+        <div className="modal-dialog" role="document">
+          <div className="modal-content ">
+            <div className="modal-header">
+              <h3 className="modal-title">
+                <strong>Login</strong>
+              </h3>
+            </div>
+            <div className="modal-body">
+              <form onSubmit={this.handleLogin}>
+                <div className="form-group">
+                  <strong>
+                    <label className="label" htmlFor="username">Username</label>
+                  </strong>
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="username"
+                    name="username"
+                    id="username"
+                    value={username}
+                    onChange={this.handleInput}
+                  />
+                </div>
+                <div className="form-group">
+                  <strong>
+                    <label className="label" htmlFor="token">API Token</label>
+                  </strong>
+                  <input
+                    id="token"
+                    type="text"
+                    className="form-control"
+                    placeholder="token"
+                    name="token"
+                    value={token}
+                    onChange={this.handleInput}
+                  />
+                </div>
+                <div className="form-group">
+                  {loading ? <Loader /> : <button
+                    type="submit"
+                    className="btn btn-login form-control"
+                  >
+                    <strong>Sign In</strong>
+                  </button>}
                 </div>
-                <div className="modal-body">
-                  <form onSubmit={this.handleLogin}>
-                    <div className="form-group">
-                      <strong>
-                        <label>Username</label>
-                      </strong>
-                      <input
-                        type="text"
-                        className="form-control"
```

Comment:
Including these, and many more?

Reply:
These changes had been effected by Hadijah when she was told to change from a modal to a normal login form, so when I rebased with master all these changes had been made.

Reply:
I see many changes that have nothing to do with what this ticket was all about. They should not be included.

Reply:
I was resolving conflicts and that is why there are some extra files in the PR.

Reply:
None of those should be in this pull request.

Reply:
I have analyzed my work and I have figured out that when I fixed the linting problems on the Login file it shows them as my changes. They are actually lint alignment issues on most of the lines in that file and fixing them led to me changing almost every line.